packages/gestalt/src/AvatarGroup.js

```diff
@@ -120,6 +120,7 @@ const AvatarGroupWithForwardRef: React$AbstractComponent<Props, UnionRefs> = for
     <CollaboratorAvatar
       hovered={hovered}
       index={index}
+      // eslint-disable-next-line react/no-array-index-key
```

Comment:
Avoiding making changes in exported components to keep the footprint of this PR small